src/components/statevector_component/aggregation.py

```diff
@@ -276,15 +420,36 @@ def update_sv_clusters(config, flat_sensi, orig_sv):
         dofs_threshold = float(config["ClusteringThreshold"])
     else:
         # default is to use the avg dofs per element
-        dofs_threshold = sum(sensitivities) / desired_num_labels
+        dofs_threshold = sum(flat_sensi) / desired_num_labels
+
+        if dofs_threshold > 1:
```

Comment:
It was happening to me, I think if `desired_num_labels` was set low. The result was that no native resolution state vector elements would be forced, because the logic for forcing native resolution in `force_native_res_pixels` is:

```python
if "ClusteringThreshold" in config.keys():
    dofs_max = float(config["ClusteringThreshold"]) + 0.1
else:
    dofs_max = 1.1
```

The better approach is to manually set a "ClusteringThreshold"

Reply:
Oh yeah, I remember now